src/post_processors/answer_score_retriever.py

```diff
@@ -0,0 +1,16 @@
+from typing import List, Tuple, Union
+
+from arqmath_code.Entities.Post import Question, Answer
+from arqmath_code.topic_file_reader import Topic
+from src.base.post_processor import PostProcessor
+
+
+class AnswerScoreRetriever(PostProcessor):
```

Comment:
Could you rename this to `AnswerScoreRetrieverForQuestions` or similar. I find the the current name confusing